media-libs/noise-suppression-for-voice/noise-suppression-for-voice-9999.ebuild

```diff
@@ -1,33 +1,42 @@
 # Copyright 2020-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
 inherit cmake
 
 DESCRIPTION="A real-time noise suppression plugin for voice"
 HOMEPAGE="https://github.com/werman/noise-suppression-for-voice"
 
-if [ "${PV}" = "9999" ]; then
+if [[ "${PV}" == "9999" ]]; then
 	inherit git-r3
 	EGIT_REPO_URI="https://github.com/werman/noise-suppression-for-voice.git"
 else
-	KEYWORDS="~amd64 ~x86"
 	SRC_URI="https://github.com/werman/noise-suppression-for-voice/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
 fi
 
 LICENSE="GPL-3+"
 SLOT="0"
-IUSE=""
-
-src_install() {
-	dodoc README.md
-
-	cd "${BUILD_DIR}"
-
-	insinto /usr/$(get_libdir)/lv2/
-	doins -r bin/rnnoise.lv2
-
-	insinto /usr/$(get_libdir)/ladspa/
-	doins bin/ladspa/librnnoise_ladspa.so
+IUSE="+ladspa lv2 vst vst3"
+
+COMMON_DEPEND="media-libs/freetype x11-libs/libX11 x11-libs/libXcursor x11-libs/libXinerama x11-libs/libXrandr"
```

Comment:
You didn't fix the newlines here?

One atom per line, so
COMMON_DEPEND="

"

Reply:
Oops, missed it due to the discussion around the deps/bdeps. Fixed.